pygments/lexers/rust.py

```diff
@@ -213,16 +213,10 @@ class RustLexer(RegexLexer):
         'attribute_common': [
             (r'"', String, 'string'),
             (r'\[', Comment.Preproc, 'attribute['),
-            (r'\(', Comment.Preproc, 'attribute('),
         ],
         'attribute[': [
             include('attribute_common'),
             (r'\];?', Comment.Preproc, '#pop'),
```

Comment:
While we are at it, I don't think Rust allows trailing semicolons after attributes so the `:?` is probably incorrect here. 
Please correct me if I'm mistaken.

Reply:
Yes, indeed.